prod-cockpit-rcar.yaml

```diff
@@ -241,6 +242,34 @@ components:
         - "tmp/deploy/images/%{MACHINE}/%{XT_XEN_DTB_NAME}"
         - "tmp/deploy/images/%{MACHINE}/core-image-minimal-%{MACHINE}.ext4"
 
+  cluster:
+    build-dir: "%{YOCTOS_WORK_DIR}"
+    sources:
+      - *COMMON_SOURCES
+      - type: git        
+        url: "git://git.yoctoproject.org/meta-arm.git"
+        rev: "f7c5e7d5094f65d105d9d580ba59527c25fb0d0f"
+      
```

Comment:
If we consider cluster as separate target, then please add required source:
      - type: git
        url: "https://github.com/xen-troops/meta-xt-prod-cockpit-rcar.git"
        rev: master